toml-benchmarks.cabal

```diff
@@ -35,7 +32,6 @@ common common-options
   if impl(ghc >= 8.8.1)
     ghc-options:       -Wmissing-deriving-strategies
                        -Werror=missing-deriving-strategies
-
   default-language:    Haskell2010
```

Comment:
All these empty lines are intentional there. No need to remove them :slightly_smiling_face: